core/src/scene/scene.cpp

```diff
@@ -70,6 +71,9 @@ Scene::Scene(const Scene& _other)
     m_globalRefs = _other.m_globalRefs;
 
     m_mapProjection.reset(new MercatorProjection());
+    for (auto& sceneAsset: _other.sceneAssets()) {
+        m_sceneAssets[sceneAsset.first.k] = sceneAsset.second;
+    }
 }
```

Comment:
Ohh I think I had a different structure for scene assets before.. hence this. But make sense with `fastmap`. Updated.